Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: all
+
+all: format build test
+
+build:
+	@cargo build 
+
+test:
+	@export RUST_BACKTRACE=1 && cargo test -- --nocapture 
+
+format: 
+	@cargo fmt -- --write-mode diff | grep -E "Diff .*at line" > /dev/null && cargo fmt -- --write-mode overwrite || exit 0
```

Comment:
Mark target `format` to be PHONY like
```Makefile
.PHONY: format
```

And output what command is about to execute for this target by running

```
make format --just-print
```

It looks like you forget to add the macro `format-code-in`

https://github.com/pingcap/rust-rocksdb/blob/hhkbp2/makefile-macro/Makefile#L18